examples/resources/dataprotection/povider.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
```

Comment:
Is there a usage for this file?

Reply:
I put this in the folder so it will be easy to test from this folder, should I remove it?